cli/step_info.go

```diff
@@ -210,12 +210,20 @@ func QueryStepInfoFromLibrary(library, id, version string) (models.StepInfoModel
 	stepDir := stepman.GetStepCollectionDirPath(route, id, stepVersion.Version)
 	stepDefinitionPth := filepath.Join(stepDir, "step.yml")
 
+	infoPath := stepman.GetStepGlobalInfoPath(route, id)
+
+	groupInfo, _, err := stepman.ParseStepGroupInfoModel(infoPath)
```

Comment:
Think should add a seperate validation that ensures that step-info.yml is valid for all of the steplib steps, or have a fallback in the case failed to parse GroupInfo. As currently QueryStepInfoFromLibrary is called on every step activate, this seems warranted.